spec/models/message_spec.rb

```diff
@@ -13,6 +13,25 @@
     end
   end
 
+  describe "criteria_rules validation" do
+    it "is valid with criteria_rules are valid" do
```

Comment:
Feels like a slight overlap of concern for the tests below compared with tests in CriteriaSchemaValidator but I suppose that's okay as this is explicitly testing the validity of the record instead of the attribute.